core/txpool/blobpool/blobpool.go

```diff
@@ -566,7 +566,7 @@ func (p *BlobPool) recheck(addr common.Address, inclusions map[common.Hash]uint6
 			ids    []uint64
 			nonces []uint64
 		)
-		for txs[0].nonce < next {
+		for ; len(txs) > 0 && txs[0].nonce < next; txs = txs[1:] {
```

Comment:
```suggestion
		for len(txs) > 0 && txs[0].nonce < next {
```
Please move the `txs = txs[1:]` back below, the code was easier to read that way.